### PR TITLE
Notes that weekend start-time varies globally

### PR DESCRIPTION
**Pull request by RobertKielty**
#### What type of PR is this:
Updates Ways of Working content in the Release Engineering README.md to remind us all that the start of the weekend varies globally.

/kind documentation

#### What this PR does / why we need it:
It's just a reminder that for many folks the weekend starts on a Thursday night and that meetings scheduled for a Friday can be problematic. 
#### Which issue(s) this PR fixes:
n/a

#### Special notes for your reviewer:

n/a

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by saschagrunert** [comment]
/hold 

for @LappleApple @justaugustus review

File: release-engineering/README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ We do not bypass the “at least one reviewer” rule, so please wait for a revi
 
 ### Communicating Your Availability: Quick Tips
 
-- We don’t require people to work on nights and weekends. However, if they’re the times you want to dedicate to the project, feel free to do so. 
+- We don’t require people to work on nights and weekends. However, if they’re the times you want to dedicate to the project, feel free to do so. It's also important to note that for some contributors the weekend starts on a Thursday, and that meetings scheduled on Friday are an inconvienience.
```
Comment (LappleApple) [inline release-engineering/README.md:101]: typo "inconvienience" ==> inconvenience

Comment (LappleApple) [reply release-engineering/README.md:101]: Thank you for this addition :)

Comment (justaugustus) [inline release-engineering/README.md:101]:
```suggestion
- We don’t require people to work on nights and weekends. However, if they are the times you want to dedicate to the project, feel free to do so. It's also important to note that for some contributors the weekend starts on a Thursday, and that meetings scheduled on Friday are an inconvenience.
```

**Comment by justaugustus** [comment]
Thanks Rob!

/lgtm
/approve
/hold cancel

(As a note for later, we should have a general "Ways of Working" for the SIG.)

File: release-engineering/README.md (hunk 1 above)
Comment (RobertKielty) [reply release-engineering/README.md]: Thank you Lauri :)

**Pull request merged**